brave/brave-ios
# Fix#2933:  Update Brave VPN intro card text + Graphic

Adding new Asset Title - Subtitle - Button Title and Graphic for VPN Intro Banner

## Summary of Changes

This pull request fixes #2933

## Submitter Checklist:


## Test Plan:

- Go to settings
- See the intro card
- Check Changes on the banner

## Screenshots:

![test33](https://user-images.githubusercontent.com/6643505/101954994-07313f80-3bcb-11eb-8c26-e01bca04a4c7.png)


## Reviewer Checklist:


  - `QA/(Yes|No)`
  - `release-notes/(include|exclude)`
  - `bug` / `enhancement`
